docs/introduction/installation.mdx

```diff
@@ -259,6 +262,19 @@ beeai env add LLM_MODEL=gpt-4o
 
 </Accordion>
 
+<Accordion title="IBM watsonx">
+
+```bash
+beeai env add LLM_API_BASE=https://us-south.ml.cloud.ibm.com
+beeai env add LLM_API_KEY=your-ibm-api-key
+beeai env add LLM_MODEL=ibm/granite-3-3-8b-instruct
+beeai env add WATSONX_PROJECT_ID=your-project-id
```

Comment:
you can use `beeai env setup` for watsonx, it will ask you for the project id or space id